src/libs/ReportActionsUtils.ts

```diff
@@ -109,6 +109,15 @@ function isChannelLogMemberAction(reportAction: OnyxEntry<ReportAction>) {
     );
 }
 
+function isReimbursementDeQueuedAction(reportID: string): boolean {
```

Comment:
Perhaps this should be called `isReimbursementDeQueued`

Reply:
spot on